info.puredata.Pd-extended.yaml

```diff
@@ -0,0 +1,117 @@
+  - --talk-name=org.freedesktop.ScreenSaver
+# MIDI access (Flatpak doesn't yet support exposing only MIDI devices)
+  - --device=all
+# Linux Audio plugins
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa:/app/lib/ladspa
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '21.08'
+    add-ld-path: lib
+    merge-dirs: ladspa
+    subdirectories: true
+    no-autodownload: true
+
+modules:
+  - shared-modules/lua5.1/lua-5.1.5.json
+#  - shared-modules/SDL/SDL-1.2.15.json # for pdp_sdl
+#  - shared-modules/SDL/SDL_image-1.2.12.json
+  - shared-modules/lame/lame-3.99.5.json
+  - shared-modules/libusb/libusb.json
+  - shared-modules/linux-audio/dssi.json
+  - shared-modules/linux-audio/fftw3f.json
+  - shared-modules/linux-audio/fluidsynth2.json
+  - shared-modules/linux-audio/jack2.json
+  - shared-modules/linux-audio/ladspa.json
+  - swig.json
+
+  - name: tcl
+    buildsystem: autotools
+    subdir: unix
+    build-options:
+      no-debuginfo: true
+    post-install:
+      - chmod 755 /app/lib/libtcl*.so
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/tcl/tcl8.6.12-src.tar.gz
+        sha256: 26c995dd0f167e48b11961d891ee555f680c175f7173ff8cb829f4ebcde4c1a6
+        x-checker-data:
+          type: anitya
+          project-id: 4941
+          stable-only: true
+          url-template: https://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz
+  - name: tk
+    buildsystem: autotools
+    subdir: unix
+    post-install:
+      - chmod 755 /app/lib/libtk*.so
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/tcl/tk8.6.12-src.tar.gz
+        sha256: 12395c1f3fcb6bed2938689f797ea3cdf41ed5cb6c4766eec8ac949560310630
+        x-checker-data:
+          type: anitya
+          project-id: 11426
+          stable-only: true
+          url-template: https://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz
+
+  - name: libtirpc
+    config-opts:
+      - --disable-gssapi
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/sourceforge/libtirpc/libtirpc-1.3.1.tar.bz2
+        sha256: 245895caf066bec5e3d4375942c8cb4366adad184c29c618d97f724ea309ee17
+
+  - name: pd-extended
+    rm-configure: true
+    sources:
+      - type: git
+        url: https://gitlab.com/pd-extended/pd-extended.git
+        commit: 39f09810b6519de0f020332db135a8845746abef
+      - type: patch
+        paths:
+          - prefix.patch
+      - type: patch
+        paths:
+          - disable-failing-externals.patch
+      - type: patch
+        paths:
+          - oscx-rpc.patch
+      - type: patch
+        paths:
+          - flatpak-package.patch
+    buildsystem: simple
+    build-commands:
+      - ./scripts/auto-build/pd-extended-release-auto-builder.sh
+    cleanup:
+      - /lib/pkgconfig
+      - /man
+
+  # AppStream metadata (for app store / package manager)
+  - name: metadata
```

Comment:
I would put that into the the "main" module, with the install command in `post-install`.

Reply:
I don't really know how to do that.

Reply:
merge the `build-commands` sections and add the `sources` to the `pd-extended` module